src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+
+      {person.isMarried === false ? (
+        `I am not married`
+      ) : `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`}
```

Comment:
You can move the second ternary operator to the separate constant:
```
const partnerStatus = person.sex === 'm' ? 'wife' : 'husband';
```
So then you can use simpler syntax:
```
: `${person.partnerName is my ${partnerStatus}`
```
It looks better